katalog/fluentd/conf.d/nginx.conf

```diff
@@ -0,0 +1,50 @@
+<filter kubernetes.var.log.containers.nginx-ingress-**.log>
```

Comment:
I am not a fluentd black belt, but can't we push the ingress controller logs. Are there any advantages to handle the logs in two different indexes?

Reply:
Is up to us.
Having it decouple make possible to configure the curator to purge indices with different configurations. 
Maybe access logs could potentially be something that you want to persist more than "regular" logs.

Reply:
+1 for having them separated

Reply:
Now I see your point and I agree with you. :-)